admin/package-files/systemd/flocker-container-agent.service

```diff
@@ -3,6 +3,7 @@ Description=Flocker Container Agent
 
 [Service]
 ExecStart=/usr/sbin/flocker-container-agent
+Restart=on-failure
```

Comment:
My impression from reading docs is that this will *not* restart if it exited cleanly with exit code 0. So perhaps we want `Restart=always` just to be safe.